docs/user_guide.md

```diff
@@ -73,70 +74,96 @@ All logs can be listed by clicking "Logs". You can apply a filter by username, o
 
-You can enable, disable or delete a policy in the policy list view. Only policies which are disabled can be edited and only policies which are disabled and have no running jobs can be deleted. If a policy is disabled, the running jobs under it will be stopped.  
+You can enable, disable or delete a rule in the rule list view. Only rules which are disabled can be edited and only rules which are disabled and have no running jobs can be deleted. If a rule is disabled, the running jobs under it will be stopped.  
 
-Click a policy, jobs which belong to this policy will be listed. A job represents the progress which will replicate a repository of one project to the remote.  
+Click a rule, jobs which belong to this rule will be listed. A job represents the progress of replicating the repository to the remote instance.  
 
-![browse project](img/new_policy_list.png)
+![browse project](img/new_rule_list.png)
 
-##Searching projects and repositories
-Entering a keyword in the search field at the top lists all matching projects and repositories. The search result includes both public and private repositories you have access privilege to.  
+## Searching projects and repositories  
+Entering a keyword in the search field at the top lists all matching projects and repositories. The search result includes both public and private repositories you have access to.  
 
 ![browse project](img/new_search.png)
 
-##Administrator options
-###Managing user
-Administrator can add "administrator" role to an ordinary user by toggling the switch under "Administrator". To delete a user, click on the recycle bin icon.  
+## Administrator options  
+### Managing user  
+Administrator can add "Administrator" role to an ordinary user by click button on the left and select "set as Administrator". To delete a user, select "Delete". 
 
 ![browse project](img/new_set_admin_remove_user.png)
 
-###Managing destination
-You can list, add, edit and delete destinations in the "Destination" tab. Only destinations which are not referenced by any policies can be edited.  
+### Managing endpoint  
+You can list, add, edit and delete endpoints in the "Endpoints" tab. Only endpoints which are not referenced by any enabled rules can be edited.  
 
-![browse project](img/new_manage_destination.png)
+![browse project](img/new_manage_endpoint.png)
 
-###Managing replication
-You can list, edit, enable and disable policies in the "Replication" tab. Make sure the policy is disabled before you edit it.  
+### Managing replication  
+You can list, edit, enable and disable rules in the "Replication" tab. Make sure the policy is disabled before you edit it.  
 
 ![browse project](img/new_manage_replication.png)
 
-##Pulling and pushing images using Docker client
+### Managing authentication
+You can change authentication mode between DB(by default) and LDAP before any user is added,when there is at least one user(besides admin) in Harbor,you cannot change the authentication mode.  

```

Comment:
You can change authentication mode between DB (default) and LDAP before any user is added. When there is at least one user (besides admin) in Harbor, you cannot change the authentication mode.